### PR TITLE
feat(api): Added http handling and Axious

### PR DESCRIPTION
**Pull request by moxort**
Added axious as a dependency,
created api folder, 
added TODOs to fix in future

### CODE REVIEW

File: frontend/src/api/AxiosInstance.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const axiosInstance = axios.create({
+    //TODO possibly change it in future to another url
+    baseURL: 'https://api.example.com',
```
Comment (LePashtet) [inline frontend/src/api/AxiosInstance.ts]: move baseURL to env variables

File: frontend/src/api/AxiosInstance.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const axiosInstance = axios.create({
```
Comment (LePashtet) [inline frontend/src/api/AxiosInstance.ts]: axiosInstance too long, usually just API is used

File: frontend/src/api/AxiosInstance.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    //TODO possibly change it in future to another url
+    baseURL: 'https://api.example.com',
+    timeout: 1000,
```
Comment (LePashtet) [inline frontend/src/api/AxiosInstance.ts]: 1 sec is too short, let's make it harder to drop and increase up to 5

File: frontend/src/api/UserApi.ts
Comment (LePashtet) [inline frontend/src/api/UserApi.ts:1]: user just enough it in api folder, right?)

**Review by LePashtet — CHANGES_REQUESTED** [review]

**Comment by LePashtet** [comment]
4 commetns added

File: frontend/src/api/UserApi.ts
Comment (moxort) [reply frontend/src/api/UserApi.ts:1]: true

**Review by LePashtet — APPROVED** [review]

**Comment by LePashtet** [comment]
Well Done!

**Pull request merged**